src/psyclone/tests/dependency_test.py

```diff
@@ -280,6 +283,45 @@ def test_goloop_partially():
            "ssh_fld%grid%tmask: READ" in str(var_accesses)
 
 
+def test_goloop_field_accesses():
+    ''' Check the handling of non-NEMO do loops.
+    TODO #440: Does not work atm, GOLoops also have start/stop as
```

Comment:
Which bit of this doesn't work? Can we have an x-failing test for it?

Reply:
That comment is incorrect here (copy&paste), I've updated the comment. There is already an x-failing test earlier.